### ISSUE TITLE
No matching distribution found for pywin32==222 on Python 3.7+3.8

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1975]
The Appveyor CI runs fail for PACKAGE_LEVEL=minimum on native Windows on Python 3.7 (for 64-bit) and 3.8 (for 32-bit and 64-bit). See https://ci.appveyor.com/project/andy-maier/pywbem/builds/29165400.

This is consistent with the package files provided for pywin32 version 222 on Pypi:

```
pywin32-222-cp27-cp27m-win32.whl
pywin32-222-cp27-cp27m-win_amd64.whl
pywin32-222-cp35-cp35m-win32.whl
pywin32-222-cp35-cp35m-win_amd64.whl
pywin32-222-cp36-cp36m-win32.whl
pywin32-222-cp36-cp36m-win_amd64.whl
pywin32-222-cp37-cp37m-win32.whl
```

The pywin32 package was added to our requirements and constraint files in PR #1947. There is no successful run of the manual-ci-run branch these Python versions since then, due to other issues, so this went by undetected up to now.

Proposal for a solution: Fix the minimum version in the requirements and constraint files, based on the following Python version support of pywin32:
* pywin32 222: py27, py35, py36, py37 (32-bit only)
* pywin32 223,224: py27, py35, py36, py37
* pywin32 225: py27, py35, py36, py37, py38, py39
* pywin32 226+: Should not be used, see issue #1946 which pins it to <=225.

**Comment by andy-maier** [comment]
Meanwhile, pywin32 has fixed the issue described in our issue #1946 in its version 227, so we should change the pinning (<=225=) to an exclusion (!=226).

**Comment by andy-maier** [comment]
PR #1976 fixes this for master, and TBD for 0.15.0.

**Comment by andy-maier** [comment]
It is more complex than I thought. An additional issue for Python 3.8 support of pywin32 is the fact that pywin32 starting with version 225 provides wheel files for py38 on Pypi, but does not advertise Python 3.8 support. This confuses pip in some cases but not in others:
* Equal constraint in minimum-constraints.txt:
  minimum-constraints.txt contains: `pywin32==225; sys_platform == 'win32' and python_version >= '3.8'`
  dev-requirements.txt contains: `pywin32>=225,!=226; sys_platform == 'win32' and python_version >= '3.8'`
  - command: `pip install -c minimum-constraints.txt -r dev-requirements.txt` -> fails with no matching distribution found
  - command: `pip install -r dev-requirements.txt` -> succeeds and installs pywin32 version 227 (the latest)
* Larger than constraint in minimum-constraints.txt:
  minimum-constraints.txt contains: `pywin32>=225,!=226; sys_platform == 'win32' and python_version >= '3.8'`
  dev-requirements.txt contains: `pywin32>=225,!=226; sys_platform == 'win32' and python_version >= '3.8'`
  - command: `pip install -c minimum-constraints.txt -r dev-requirements.txt` -> fails with no matching distribution found
  - command: `pip install -r dev-requirements.txt` -> succeeds and installs pywin32 version 227 (the latest)
* No constraint in minimum-constraints.txt:
  minimum-constraints.txt contains no specification for pywin32 for sys_platform == 'win32' and python_version >= '3.8'
  dev-requirements.txt contains: `pywin32>=225,!=226; sys_platform == 'win32' and python_version >= '3.8'`
  - command: `pip install -c minimum-constraints.txt -r dev-requirements.txt` -> succeeds and installs pywin32 version 227 (the latest)
  - command: `pip install -r dev-requirements.txt` -> succeeds and installs pywin32 version 227 (the latest)

**Comment by andy-maier** [comment]
Addressed along with other issues, in PR #1976 for master, and TBD for 0.15.0.
